src/components/App/App.js

```diff
@@ -1,14 +1,11 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import './App.css';
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        {this.props.children}
-      </div>
-    );
-  }
-}
+const App = ({ children }) => <div className="App">{children}</div>;
```

Comment:
👍

Reply:
Agreed, definitely prefer the functional style